src/modules/dashboard/components/TaskEditDialog/WrappedPayout.tsx

```diff
@@ -1,19 +1,16 @@
 import React, { useCallback } from 'react';
 
-import {
-  TaskPayoutType,
-  TaskType,
-  ColonyTokenReferenceType,
-} from '~immutable/index';
+import { TaskPayoutType, ColonyTokenReferenceType } from '~immutable/index';
 import { Address } from '~types/index';
 import Payout from './Payout';
 
-interface Props extends Pick<TaskType, 'payouts' | 'reputation'> {
+interface Props {
   arrayHelpers: any;
   canRemove: boolean;
   colonyAddress: Address;
   index: number;
   payout: TaskPayoutType;
+  payouts: TaskPayoutType[];
```

Comment:
Can you add a // FIXME (or @todo) here to replace this with a type on the server?